Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.8-alpine
```

Comment:
I would, if possible, allow people to configure the image with something like `FROM ${image_name}`